src/python/WMQuality/Emulators/SiteDBClient/MockSiteDBApi.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python
+
+"""
+Version of SiteDB.SiteDBJSON intended to be used with mock or unittest.mock
+"""
+from __future__ import (division, print_function)
+
+from WMCore.WMBase import getTestBase
+from RestClient.ErrorHandling.RestClientExceptions import HTTPError
+import os
+import json
+
+# Read in the data just once so that we don't have to do it for every test (in __init__)
+mockData = {}
+globalFile = os.path.join(getTestBase(), '..', 'data', 'Mock', 'SiteDBMockData.json')
+print(globalFile)
+
+try:
+    with open(globalFile, 'r') as mockFile:
+        mockData = json.load(mockFile)
+except IOError:
+    mockData = {}
+
+
+def mockGetJSON(dummySelf, callname, filename='result.json', clearCache=False, verb='GET', data=None):
+    """
+    _getJSON_
```

Comment:
Put a real doc string here that describes what you are doing. We certainly don't want to move this into Service. :-)